salt/metalk8s/saltmaster/files/master_99-metalk8s.conf.j2

```diff
@@ -0,0 +1,9 @@
+file_roots:
+  metalk8s-{{ metal_version }}:
+    - /srv/scality/metalk8s-dev/salt
```

Comment:
Need a tracking ticket for this hard-coded path (cfr. #639)

Reply:
@sayf-eddine-scality has the ticket been created?

Reply:
#651